realm/src/main/java/io/realm/internal/SharedGroup.java

```diff
@@ -164,6 +168,23 @@ void endRead() {
         activeTransaction = false;
     }
 
+    public void beginReadAtVersionID (long [] versionID) {
+        nativeBeginReadAtVersionID(nativePtr, versionID);
+    }
+
+    private native void nativeBeginReadAtVersionID (long nativePtr, long [] versionID);
+
+    /**
+     * Return the current SharedGroup VersionID, this is called for example
+     * to position a Realm to a specific version.
+     * @return array of size 2, position 0 is the 'uint_fast64_t version' position 1 is the 'uint_fast32_t index'
```

Comment:
Maybe add that both are needed when returning this value to core. At least I was a bit confused by this.